benchmark_runner/common/ocp_resources/create_odf.py

```diff
@@ -42,7 +42,7 @@ def create_odf(self, upgrade_version: str):
                         for node, disk_ids in self.__worker_disk_ids.items():
                             for disk_id in disk_ids:
                                 disk = f'/dev/disk/by-id/{self.__worker_disk_prefix}{disk_id}'
-                                delete_node_disk += f"""sudo sgdisk --zap-all {disk}; sudo wipefs -a {disk}; sudo dd if=/dev/zero of='{disk}' bs=1M count=100 oflag=direct,dsync; sudo blkdiscard {disk}; sudo partprobe {disk};"""
+                                delete_node_disk += f"""sudo sgdisk --zap-all {disk}; sudo wipefs -a {disk}; sudo dd if=/dev/zero of='{disk}' bs=1M count=100 oflag=direct,dsync; """
```

Comment:
I was thinking about something like this. so we can check if any of the command executed succesfully or not and which disk it failed operating on. 

`sudo sgdisk --zap-all {disk} && echo "✓ GPT/MBR zapped on {disk}" || echo "✗ GPT/MBR zap failed on {disk}"; sudo wipefs -a {disk} && echo "✓ Signatures wiped on {disk}" || echo "✗ Signature wipe failed on {disk}"; sudo dd if=/dev/zero of='{disk}' bs=1M count=100 oflag=direct,dsync && echo "✓ Zeroing successful on {disk}" || echo "✗ Zeroing failed on {disk}"`

Reply:
No need to add echo inside the code. I will print the output of the whole shell, so if there is any error, we will see it while running.